packages/account-v2/src/containers/LoginHistoryTable/LoginHistoryTableCard.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { LoginHistory } from '@deriv/api-types';
+import { Text } from '@deriv-com/ui';
+import { formattedLoginHistoryData } from '../../utils';
+
+type TLoginHistoryProps = { loginHistory: LoginHistory };
+export const LoginHistoryTableCard = ({ loginHistory }: TLoginHistoryProps) => {
+    const formattedLoginHistory = formattedLoginHistoryData(loginHistory);
+    return (
+        <div className='inline-flex flex-col items-start gap-16'>
+            {' '}
+            {/* Loop through formatted login history data */}{' '}
+            {formattedLoginHistory.map((item, index) => (
+                <div
+                    className='shadow-md p-16 rounded-lg background: var(--system-light-8-primary-background, #FFF)'
```

Comment:
```suggestion
                     background: var(--system-light-8-primary-background, #FFF)
```

Incorrect tailwind syntax

Reply:
could u clarify the syntax error as if i remove `shadow-md p-16 rounded-lg` the bg of the card goes and it comes wthout a shadow

Reply:
`background: var(--system-light-8-primary-background, #FFF)` this is incorrect